### PR TITLE
item #6527 updates for Ldap multi-auth test automation

### PR DESCRIPTION
**Pull request by labkey-chrisj**
adds support for multi-auth/ldap automation

related PRs in: ldap module

### CODE REVIEW

File: src/org/labkey/test/pages/core/login/LoginConfigurePage.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        return this;
+    }
+
+    public LdapConfigurePage clickEditLdapConfiguration(String description)
+    {
+        WebElement row = Locator.xpath("//tbody/tr").containing(description).findElement(elementCache().tableElement);
+        clickAndWait(Locator.tagWithClass("a", "labkey-text-link").withText("delete").findElement(row));
+
+        return new LdapConfigurePage(getDriver());
+    }
```
Comment (labkey-tchad) [inline src/org/labkey/test/pages/core/login/LoginConfigurePage.java]: Remove LDAP-specific methods.

Comment (labkey-chrisj) [reply src/org/labkey/test/pages/core/login/LoginConfigurePage.java]: good call

**Comment by labkey-chrisj** [comment]
tests passing on TC
https://teamcity.labkey.org/viewLog.html?buildId=866366&buildTypeId=LabkeyTrunk_GitPostgres&tab=testsInfo

**Comment by labkey-tchad** [comment]
It is my understanding that the UI design for this is undergoing some major changes. I think the authentication provider configuration is moving from separate actions to a modal dialog with no navigation. We should wait until the redesign is complete.

**Comment by labkey-chrisj** [comment]
Yes, the plan is to update the UI to React.  My conversations with Adam left me with the expectation that he wanted to merge this now, and let Rosaline finish her work on top of the new Ldap implementation.

**Comment by labkey-adam** [comment]
Yes, the LDAP product changes have been done for a couple weeks and are waiting for the test changes. We finally have an LDAP authentication test, yay! Sounds deliverable to me. In general, we want to deliver incrementally, even when future refactors are anticipated (or even in progress on other branches). In this specific case, Rosaline's changes are actually blocked on this merge -- e.g., she needs to make changes to the new LDAP module. The test flow and vast majority of server-side functionality will remain the same... the only change is the way the settings forms are rendered.

**Pull request merged**